packages/client/mutations/handlers/handleRemoveTasks.ts

```diff
@@ -24,13 +24,13 @@ const handleRemoveTask = (taskId: string, store: RecordSourceSelectorProxy<any>)
   const meetingId = task.getValue('meetingId')
   const meeting = store.get(meetingId!)
   const team = store.get(teamId)
-  const archiveConn = getArchivedTasksConn(viewer, teamId)
+  const archiveConns = [getArchivedTasksConn(viewer, teamId), getArchivedTasksConn(viewer)]
```

Comment:
instead of adding another archiveConn would it be possible to add these params to the userConn & make the archiveConns go away?

e.g. `ConnectionHandler.getConnection(viewer, 'UserTasks_tasks', {userId, teamId, isArchived})`